zynlibs/jackpeak/jackpeak.c

```diff
@@ -70,7 +70,7 @@ int initJackpeak() {
 
 	// Register the cleanup function to be called when program exits
 	// It raises a segmentation fault on exit, so it's disabled
-	//atexit(endJackpeak);
+	atexit(endJackpeak);
```

Comment:
DId you check if the segmentation fault is still there when exiting the UI?

Reply:
I have committed a change that removes the comment and actually runs the exit code. Segmentation fault is resolved.